server/main/templates/account/login.html

```diff
@@ -5,6 +5,13 @@
 {% load branding %}
 {% load static %}
 
+{% block head_title %}
+  {% trans 'Experiment Data Depot' context 'HTML Title' %}
```

Comment:
Should use `{{ block.super }}` to inherit from the base template, instead of duplicating the content. [Extending an overridden template](https://docs.djangoproject.com/en/dev/howto/overriding-templates/#extending-an-overridden-template-1)